uber/templates/static_views/js/prereg-common.js

```diff
@@ -0,0 +1,183 @@
+var PREREG_CHECK_INTERVAL = 15 * 60 * 1000;
+var checkCap = function () {
+    $.getJSON('../preregistration/check_prereg', function (check) {
+        if (check.force_refresh) {
+            location.reload();  // Reload the page to prevent registering after reg is closed
+        } else {
+            setTimeout(checkCap, PREREG_CHECK_INTERVAL);
+        }
+    });
+};
+setTimeout(checkCap, PREREG_CHECK_INTERVAL);
+
+{% if c.COLLECT_FULL_ADDRESS %}
```

Comment:
This increases page weight - I'd rather find a way to construct one file with all the relevant pieces in it.

> On Jun 15, 2016, at 5:15 AM, Victoria Earl <notifications@github.com> wrote:
> 
> For these simple checks, is it worth it to put the JS in its own file and put the check in the template?